Makefile

```diff
@@ -108,6 +108,26 @@ clean:
         clientapi/webrca \
 		$(NULL)
 
+.PHONY: tag
+tag:
```

Comment:
How do we validate that the tag is created from the `main` branch?

Reply:
[[ "$(git symbolic-ref --short HEAD 2>/dev/null)" == "main" ]] && echo "On main branch" || echo "Not on main branch"